### ISSUE TITLE
https://mitorrent.org/peliculas/rango/

### ISSUE THREAD

**Issue by LinkColt** [Sainan/Universal-Bypass#2115]
When you click the download link "1080 dual latino" (or click "Descargar Latino" and then "Descargar"), you get redirected like 2 times to a page called "acortaenlace.com" (the culprit) with nothing else in the url, that's why I didn't paste that URL instead, cause it seems that it's generated individually every time you click on the "Descargar" button so it would lead to nowhere, that's all the info I can provide with my very limited knowledge about this stuff, hopefully you guys can figure it out once you see it.

So after you get to that sketchy url shortener page you have to follow some steps to unlock the actual download link, when I've done it I had to search for a news website (the sponsor I assume) on Google then click on a result and stay on that page for like 40 seconds.

If you guys have any questions or need help translating I will be happy to help :)

**Comment by LinkColt** [comment]
I know this is obvious but I would like to point out that this does not only happen on this particular movie, it's in all of them but I chose this one as an example so you guys can check out that weird shapeshifting link

**Comment by UnJust1ce** [comment]
Hey, just letting you know, universal bypass has been discontinued (no longer updated, and crowd bypass servers are down). You should probably move to https://fastforward.team (the official continuation) and report the link there.

**Comment by LinkColt** [comment]
yes thank u, I found out a couple days after posting this, I opened an issue a while back and there's still no responses

**Comment by UnJust1ce** [comment]
Sorry about that, the GitHub page for fast forward is not looked at by the devs very often but imma go through and add all the bypasses very soon

**Comment by LinkColt** [comment]
thank u :)
